docs/Users/Install.rst

```diff
@@ -153,6 +153,18 @@ If you want to install coala to an alternate location you can e.g. call
 options are documented on
 https://docs.python.org/3.3/install/#alternate-installation.
 
+.. note::
+
+    If you are using a proxy, follow these steps:
+
+    -  Set up your proxy.
+    -  Use ``sudo -E pip install coala``(-E flag takes in the existing
```

Comment:
When you say takes in, the first thing I think of is `-E <some_argument>`, could you say something else, like `preserves the existing environment variables`